### PR TITLE
Token Binding Sample

### PR DESCRIPTION
**Pull request by inthirakumaaran**
This contains the sample application for token binding

### CODE REVIEW

File: modules/samples/tokenBinding/layout/js/npm.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+require('../../js/button.js')
+require('../../js/carousel.js')
+require('../../js/collapse.js')
+require('../../js/dropdown.js')
+require('../../js/modal.js')
+require('../../js/tooltip.js')
+require('../../js/popover.js')
+require('../../js/scrollspy.js')
+require('../../js/tab.js')
+require('../../js/affix.js')
```
Comment (Kanapriya) [inline modules/samples/tokenBinding/layout/js/npm.js]: Add new line here

File: modules/samples/tokenBinding/authorization.php [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    curl_close($curl);
+
+    if ($err) {
+        echo "cURL Error #:" . $err;
+    } else {
+        echo $response;
+    }
+}
+else{
```
Comment (Kanapriya) [inline modules/samples/tokenBinding/authorization.php]: Formatting issue, could you please check the formatting throughput the PR?

File: modules/samples/tokenBinding/callback.php [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php include "/opt/lampp/htdocs/password/layout/header.php"; ?>
+<?php
+$code = $_GET['code'];
+echo "<h5>" . $code . "</h5>";
+
+?>
+
+
```
Comment (Kanapriya) [inline modules/samples/tokenBinding/callback.php]: Remove unwanted new lines

File: modules/samples/tokenBinding/layout/js/npm.js (hunk 1 above)
Comment (inthirakumaaran) [reply modules/samples/tokenBinding/layout/js/npm.js]: PR send https://github.com/wso2/product-is/pull/6870

File: modules/samples/tokenBinding/authorization.php (hunk 2 above)
Comment (inthirakumaaran) [reply modules/samples/tokenBinding/authorization.php]: PR send https://github.com/wso2/product-is/pull/6870

File: modules/samples/tokenBinding/callback.php (hunk 3 above)
Comment (inthirakumaaran) [reply modules/samples/tokenBinding/callback.php]: PR send https://github.com/wso2/product-is/pull/6870

**Pull request merged**